server/public/styles.css

```diff
@@ -0,0 +1,32 @@
+* {
+  font-family: Nunito, sans-serif, serif;
+}
+
+p,
+span {
+  font-family: "Open Sans", sans-serif, serif;
+}
+
+.open-sans-text {
+  font-family: "Open Sans", sans-serif, serif;
+}
```

Comment:
I'm not super sure about this font styling. Typically you want to style `body` instead of `*` to apply fonts, and just let inheritance take over. Inputs and buttons can be explicitly targeted with a group element selector

```css
body {
  font-family: your default font
}

h1,
h2,
etc {
  font-family: your heading font
}

/* these are not inherited */

input,
button {
  font-family: your form control font
}
```